main/src/test/scala/org/clulab/numeric/TestSeasonNormalizer.scala

```diff
@@ -58,4 +63,36 @@ class TestSeasonNormalizer extends Test {
     seasonNorms shouldNot contain (fallDateRange)
     seasonNorms should contain (seasonDateRange)
   }
-}
+
+  behavior of "Default SeasonalCluProcessor"
+
+  it should "find true seasons in trueSeason1" in {
+    val processor = new CluProcessor()
```

Comment:
> There should probably be a single `CluProcessor` for the entire file of tests. It is very expensive to create. If the processor above is moved like so, it might work:
> 
> ```scala
>   behavior of "Default SeasonalCluProcessor"
> 
>   val processor = new CluProcessor()
> 
>   it should "find autumn but not rainy season" in {
> ```

Thanks for the great suggestion! I didn't know if that's possible or not (and also didn't know that creating `CluProcessor` is very expensive!) so just followed the pattern I found from the other test. I'll change the code as you suggested and see how it works.